### ISSUE TITLE
Icon request: icon-na

### ISSUE THREAD

**Issue by jenniferzint** [FortAwesome/Font-Awesome#9025]
I am using square-o and check-square-o in a company form. The existing form items needed only checked/unchecked responses, but I need to create a new section that also allows for an N/A option. Would love an icon similar to square-o with the letters NA or N/A inside to fulfill this need. 

Thank you!
Jennifer

**Comment by jennz123** [comment]
something like this...
![not_applicable](https://cloud.githubusercontent.com/assets/18576294/14678874/2039581e-06dd-11e6-9364-fddfb1b8dab5.png)

**Comment by tagliala** [comment]
NA stands for Not Available, am I right?

**Comment by jennz123** [comment]
Yes it does.

**Comment by tagliala** [comment]
Does [fa-ban](http://fontawesome.io/icon/ban/) fit your use case?

**Comment by Zopieux** [comment]
What about, you know, using the letters N and A with some CSS? The example icon you provided looks like a abbreviated United States state name or a badly cased sodium element.

**Comment by tagliala** [comment]
@Zopieux I agree that this could be the case of using plain CSS. Also, N/A is not localizable

**Comment by jennz123** [comment]
fa-ban would be the closest thing to work, but it is not ideal. The existing form is completed mostly on phone devices by field techs. The new section contains items that some field techs do not have so I would like to use the N/A icon in those cases, simply bc it would be more user friendly and that is imperative w/ this set of users if you catch my drift. lol. 

Regarding using regular text, it is a job safety inspection form and there are a LOT of items. So, I am querying the db for the items and their given value and displaying for them. If they select that a particular item has an issue, the value updates and icon class changes to checked and they are required to fill out a remark on the next section. I am tallying items with a certain icon class to determine if a remark is needed. The NA icon would be ideal because it would allow me to keep the programming that is already in place and just make some slight changes, but if it is not possible, I will rewrite the way the app works.

**Comment by jennz123** [comment]
I just used radio buttons for the new section and revised the jquery to tally in a different way. Thanks anyway.

**Comment by stekkedoos** [comment]
i would love this kind of icon as well , could be used as "not applicable" as well

**Comment by abrambailey** [comment]
I was also looking for an N/A icon.

**Comment by cwolcott** [comment]
I was also hoping to find an icon like this

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
